doc/en/announce/release-5.3.3.rst

```diff
@@ -0,0 +1,30 @@
+pytest-5.3.3
+=======================================
+
+pytest 5.3.3 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/latest/changelog.html.
+
+Thanks to all who contributed to this release, among them:
+
+* Adam Johnson
+* Alexandre Mulatinho
+* Anthony Sottile
+* Bruno Oliveira
+* Chris NeJame
+* Daniel Hahler
+* Hugo
```

Comment:
```suggestion
* Hugo van Kemenade
```

Reply:
@hugovk 
You might want to fix/adjust your Git config then maybe (it comes via 8de9b1b).